osi_trafficlight.proto

```diff
@@ -70,6 +70,11 @@ message TrafficLight
         //
         repeated Identifier assigned_lane_id = 5;
 
+        // Boolean flag to indicate that the traffic light is taken out of service.
+        // This can be achieved by visibly crossing the light or covering it completely. 
```

Comment:
I'd propose to add that the traffic light might also be turned off.

Reply:
Good point. This is fixed now.